files/system/usr/share/secureblue/install_secureblue.sh

```diff
@@ -46,7 +46,7 @@ echo "After answering the following questions, your system will be rebased to se
 echo "-----------------------------------------------------------------------------------"
 
 # Determine if it's a server or desktop
-read -p "Is this for a server? (yes/No): " is_server
+read -p "Is this for a CoreOS server? (yes/No): " is_server
```

Comment:
we should split these scripts since the coreos instructions will be significantly different